src/main/res/values/strings.xml

```diff
@@ -164,20 +164,20 @@
     <string name="upstream_source_code_is_not_free">The upstream source code is not free</string>
     <string name="username">Username</string>
     <string name="username_missing">Username missing</string>
-    <string name="validation_index_error_DESC">Index could not be validated.</string>
+    <string name="validation_index_error_DESC">Could not validate index.</string>
     <string name="version">Version</string>
     <string name="version_FORMAT">Version %s</string>
     <string name="versions">Versions</string>
-    <string name="waiting_to_start_download">Waiting to start download…</string>
+    <string name="waiting_to_start_download">Starting download…</string>
     <string name="whats_new">What\'s New</string>
     <string name="website">Website</string>
     <string name="prefs_language_title">Language</string>
     <string name="prefs_personalization">Personalization</string>
     <string name="show_less">Show Less</string>
     <string name="latest">Latest</string>
-    <string name="explore">Explore</string>
-    <string name="update_all">Update all</string>
-    <string name="installed_applications">Installed applications</string>
-    <string name="sort_filter">Sort &amp; Filter</string>
-    <string name="new_applications">New applications</string>
+    <string name="explore">Explore…</string>
+    <string name="update_all">Upgrade all</string>
```

Comment:
?

Reply:
Update would be more appropriate according to me

Reply:
The dichotomy is avoided in all other strings, so maybe
"Install all new versions"?

Reply:
```suggestion
    <string name="update_all">Install all new versions</string>
```

Reply:
Would suggest a concise one because this will be used in fab in future is currently not used anywhere

Reply:
This fits?

Reply:
```suggestion
    <string name="update_all">Install all</string>
```